application/forms/PreferenceForm.php

```diff
@@ -283,6 +294,36 @@ public function createElements(array $formData)
             )
         );
 
+        if (isset($formData['auto_refresh']) && $formData['auto_refresh']) {
+            $speeds = [1 => $this->translate('Default')];
+
+            foreach ([2, 4, 8] as $speed) {
+                // Using Form#translatePlural() not for $speed==1 and $speed!=1,
+                // but for different $speed-dependent plural forms, e.g. in Russian
+                $speeds[$speed] = sprintf($this->translatePlural('%dx slower', '%dx slower', $speed), $speed);
+                $speeds[rtrim(sprintf('%F', 1.0 / $speed), '0')] = sprintf(
+                    $this->translatePlural('%dx faster', '%dx faster', $speed),
+                    $speed
+                );
+            }
+
+            krsort($speeds);
+
+            $this->addElement(
+                'select',
+                'auto_refresh_speed',
+                array(
+                    'required'      => false,
+                    'label'         => $this->translate('Auto refresh speed'),
+                    'description'   => $this->translate(
+                        'This option allows you to speed up or to slow down the global page content auto refresh'
+                    ),
+                    'multiOptions'  => $speeds,
+                    'value'         => 1
```

Comment:
Use the empty string here as default value. This results in the preference setting being removed from the configuration once the user chooses `Default`.